src/ActiveField.php

```diff
@@ -383,6 +386,7 @@ public function radioList($items, $options = [])
                     Html::addCssClass($wrapperOptions, 'custom-control-inline');
                 }
 
+                $this->addErrorClassIfNeeded($options);
```

Comment:
Should it be here or later on (before line 401)?

Reply:
It should be here, radioList contains several values and should be added here for correct display.

Reply:
389 line